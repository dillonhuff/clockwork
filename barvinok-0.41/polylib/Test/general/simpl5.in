# {i,j,k | k=0; 1<=i<=10; 1<=j<=10} |
# {i,j,k | k=0; 28<=i<=31; 1<=j<=10}
D 2
5 5
0 0 0 1 0
1 1 0 0 -1
1 -1 0 0 10
1 0 1 0 -1
1 0 -1 0 10
5 5
0 0 0 1 0
1 1 0 0 -28
1 -1 0 0 31
1 0 1 0 -1
1 0 -1 0 10
# {i,j,k | k=0; i+j>=25; i<=25; j<=25} |
# {i,j,k | k=0; 2<=j<=9; 30<=i<=32}
D 2
4 5
0 0 0 1 0
1 1 1 0 -25
1 -1 0 0 25 
1 0 -1 0 25
5 5
0 0 0 1 0
1 0 1 0 -2
1 0 -1 0 9
1 1 0 0 -30
1 -1 0 0 32
F 2

DomSimplify[readDom["{i,j,k | k=0; 1<=i<=10; 1<=j<=10} |
                     {i,j,k | k=0; 28<=i<=31; 1<=j<=10}"],
            readDom["{i,j,k | k=0; i+j>=25; i<=25; j<=25} |
                     {i,j,k | k=0; 2<=j<=9; 30<=i<=32}"]]
(*answer =
domain[3, {i, j, k}, {pol[2, 4, 0, 2,
    {{1, 1, 0, 0, -28}, {1, -1, 0, 0, 31}},
    {{0, 0, 1, 0, 0}, {0, 0, 0, 1, 0}, {1, 28, 0, 0, 1}, {1, 31, 0, 0, 1}}]}]
{i,j,k|i-28>=0;-i+31>=0}
*)
