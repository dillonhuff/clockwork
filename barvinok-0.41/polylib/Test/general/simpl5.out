2 5
   1    1    0    0  -28 
   1   -1    0    0   31 
